int64 utcTime
int32 latitude
int32 longitude
int16 elevation
uint32 accuracy
uint16 speed
uint16 heading
